# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../i2c-tools/i2c-tools-3.1.0-no-install-kernel-header.patch
# Copyright (C) 2012 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- i2c-tools-3.1.0/include/Module.mk.orig	2012-09-07 18:11:23.621081505 +0200
+++ i2c-tools-3.1.0/include/Module.mk	2012-09-07 18:12:02.909755477 +0200
@@ -24,6 +24,6 @@
 	for file in $(INCLUDE_TARGETS) ; do \
 	$(RM) $(DESTDIR)$(incdir)/$$file ; done
 
-install: install-include
+install:
 
-uninstall: uninstall-include
+uninstall:
